src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public Excavator(String nameMachine) {
+        super(nameMachine);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(getNameMachine() + " start work");
```

Comment:
The message in the 'doWork' method is not informative enough. It should indicate that an 'Excavator' is working. Consider changing the message to include the type of the machine, for example: 'Excavator ' + getNameMachine() + ' started working.'